content/100-getting-started/02-setup-prisma/200-add-to-existing-project/120-mongodb/index.mdx

```diff
@@ -22,8 +22,9 @@ In order to successfully complete this guide, you need:
 - Access to a MongoDB 4.2+ server with a replica set deployment. We recommend using [MongoDB Atlas](https://www.mongodb.com/cloud/atlas).
 
   <Admonition type="warning">
-  
- The MongoDB database connector uses transactions to support nested writes. Transactions **requires** a [replica set](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) deployment. The easiest way to deploy a replica set is with [Atlas](https://docs.atlas.mongodb.com/getting-started/). It's free to get started.
+
+  The MongoDB database connector uses transactions to support nested writes. Transactions **requires** a [replica set](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) deployment. The easiest way to deploy a replica set is with [Atlas](https://docs.atlas.mongodb.com/getting-started/). It's free to get started.
```

Comment:
Looks like it fixed markup in https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/mongodb-typescript-mongodb#prerequisites

It's looks like this now

<img width="712" alt="image" src="https://user-images.githubusercontent.com/4734297/125611379-53b29429-286c-4e58-80f6-27918092ef80.png">